tests/features/change_tracking_workflow.feature

```diff
@@ -0,0 +1,170 @@
+
+ @api
+  Scenario Outline: If no changing tracks exist, I do not see any messages or HTML tags related to the change tracking
+    Given I am logged in as a user with the 'administrator' role
+    When I go to "admin/config/content/multisite_wysiwyg/workbench"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-validated"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-published"
+    And I press "Save configuration"
+    When I am viewing an "page" content:
+      | title            | Lorem ipsum dolor sit amet                                      |
+      | body             | <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p> |
+      | moderation state | draft                                                           |
+    When I click "Edit draft"
+    And I fill in "Body" with "<html>"
+    And I press "Save"
+    Then the response should contain "<expected>"
+    And I should not see the following warning messages:
+      | warning messages | <strong>The change tracking is activated on some fields of this "Basic page" content</strong>.<br /> <small>Please accept or reject tracked changes before setting the content state to validated or published.</small> |
+    And I should not see an "div.ICE-Tracking" element
+
+    Examples:
+      | html                                                                                                       | expected                                                                                                   |
+      | <p>No ice-ins or ice-del tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p> | <p>No ice-ins or ice-del tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p> |
+      | <p>No tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p>                    | <p>No tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p>                    |
+
+  @api
+  Scenario Outline: The change of the content state to "validated" or "published" must be blocked if
+  CKEditor Lite tracked changes exist in WYSIWYG fields of a translation
+    Given I am logged in as a user with the 'administrator' role
+    And the following languages are available:
+      | languages |
+      | en        |
+      | fr        |
+    When I go to "admin/config/content/multisite_wysiwyg/workbench"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-validated"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-published"
+    And I press "Save configuration"
+    Then I should see the success message "The configuration options have been saved."
+    And I am viewing a multilingual "page" content:
```

Comment:
Use When here because the previous line is a Then, but this line does not verify an observable outcome